colaraz/cms/static/sass/shared-v2/_dashboard.scss

```diff
@@ -139,3 +139,7 @@
         }
     }
 }
+
+.new-library-org {
+    width: 100%;
+}
```

Comment:
If there is any other parent selector for this selector, please put this rule there to avoid global rules.

Reply:
This class is only being applied to this  place only. So it will not interfere with other global rules.